.claude/commands/debug-protocol.md

```
@@ -75,61 +74,61 @@ Using Serena MCP for semantic code analysis:
 mcp__serena__get_symbols_overview --relative_path "[directory]"
 ```
-- Identify equivalent functions/components between systems (e.g., V1 resumeCampaign vs V2 GamePlayView)
-- Map corresponding API endpoints, data handlers, UI components
-- Document component relationships and dependencies
+10. Identify equivalent functions/components between systems (e.g., V1 resumeCampaign vs V2 GamePlayView)
+11. Map corresponding API endpoints, data handlers, UI components
+12. Document component relationships and dependencies
 
 **Step 2: Side-by-Side Code Analysis (10-15 minutes)**
 Using systematic comparison methodology:
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)